ios/RNWalletModule/RNWalletModule.m

```diff
@@ -29,13 +29,22 @@ @implementation RNWalletModule
 }
 
 RCT_EXPORT_METHOD(
-	showAddPassController:(NSString *)pass
+                  showAddPassControllerFromData:(NSData *)data
```

Comment:
Doesn't seem as ReactNative supports NSData as a parameter:
https://facebook.github.io/react-native/docs/native-modules-ios.html#argument-types

Reply:
tks, we'll change

Reply:
```
After further research into the source code I found that you can't declare an NSData argument, you have to pass it in as a base64 encoded string (NSString) and then use RCTConvert to convert it to an NSData.
```
```
RCT_EXPORT_METHOD(write:(CBUUID *)peripheralUuid 
                  forCharacteristicUuid:(CBUUID *)characteristicUuid 
                  data:(NSString *)data
                  withResponse:(BOOL)withResponse)
{
    // ....
    [peripheral 
          writeValue:[RCTConvert NSData:data] 
          forCharacteristic:/* ... */
          type:/* ... */];
}
```

this is easily fixable, we'll change it.